web-server/v0.4/src/pages/Dashboard/RunComparison.js

```diff
@@ -221,7 +221,7 @@ export default class RunComparison extends ReactJS.Component {
           iterationRequests.push(
             axios.get(
               datastoreConfig.results +
-              '/results/' +
+              '/incoming/' +
```

Comment:
The incoming endpoint fetches empty response data for the timeseries graphs under the 'All' tab in the run comparison page. This ends up rendering empty timeseries charts within the tab. Do we have the result.json files for sample data at this endpoint?

Reply:
@gurbirkalsi is this for all result data?  The `/results/` path ends up being a link to the `/incoming/` path, so if you have a particular example I can debug what is going on.

Reply:
For some reason my browser was blocking the resources earlier - it worked once I reset my CORS extension.